src/use-cases/enrichHearingOutcome/enrichFunctions/enrichOffences/enrichOffence.ts

```diff
@@ -15,7 +15,7 @@ const enrichOffence = (offence: Offence): Offence => {
 
   if (offenceIgnored) {
     offence.OffenceCategory = "B7"
-    offence.RecordableOnPNCindicator = "N" // other possible options are "D" (don't know), "NA" (not applicable)
+    offence.RecordableOnPNCindicator = false // other possible options are "D" (don't know), "NA" (not applicable)
```

Comment:
If `RecordableOnPNCindicator` has other possible options, does it belong as an enum instead?

Reply:
Looks like we only use it as a boolean(Y or N) in the core repo, not sure about old Bichard 😕

Reply:
Removed comment in [d043f17](https://github.com/ministryofjustice/bichard7-next-core/pull/72/commits/d043f17a672fa22ed44b4cdd4840e28ceac0bb49)